.travis.yml

```diff
@@ -65,21 +66,24 @@ before_install:
   fi
 
 install:
-- conda install --yes python=$PY numpy==1.11.2 scipy nose sphinx mock swig
-- if [ "$OS" = "Linux" ]; then
-    if [ "$PY" = "2.7" ]; then
-      pip install https://openmdao.org/dists/pyoptsparse-1.0.0-cp27-none-linux_x86_64.whl;
-    elif [ "$PY" = "3.4" ]; then
-      pip install https://openmdao.org/dists/pyoptsparse-1.0.0-py3-none-linux_x86_64.whl;
-    fi
+# keeping numpy pinned at 1.12 because snopt dies with 1.13 because numpy broke callback
+- conda install --yes python=$PY numpy==1.12 scipy nose sphinx mock swig
+
+# trying a manual install of pyoptsparse
+- if [ "$PY" = "2.7" ]; then
+    pip install mercurial;
+    hg clone https://bitbucket.org/mdolab/pyoptsparse;
+    cd pyoptsparse;
+    python setup.py install;
+    cd ..;
   fi
 
-- if [ "$OS" = "MacOSX" ]; then
-    if [ "$PY" = "2.7" ]; then
-      pip install https://openmdao.org/dists/pyoptsparse-1.0.0-py2-none-macosx_10_5_x86_64.whl;
-    elif [ "$PY" = "3.4" ]; then
-      pip install https://openmdao.org/dists/pyoptsparse-1.0.0-cp34-none-macosx_10_5_x86_64.whl;
-    fi
+- if [ "$PY" = "3.6" ]; then
+    wget https://bitbucket.org/mdolab/pyoptsparse/get/9af3264429b9.zip;
+    unzip 9af3264429b9.zip;
+    cd mdolab-pyoptsparse-9af3264429b9;
+    python setup.py install;
+    cd ..;
```

Comment:
wonder if it would be better to use a this method for both versions...  the clone will mean we are always testing the latest pyoptsparse on py2.7, so I guess we should be explicit that that is what we want

Reply:
I could try to do the same setup and do "tip" on both...let me try that and see if it fails miserably?  OR would you rather they are both pinned to a stable commit?

Reply:
I do think it would be cleaner to do it one way for both versions of python... either with `tip` or the specific commit, depending on the consensus

Reply:
I think we'll use "tip." Just for a few commits, just to see what it feels like.